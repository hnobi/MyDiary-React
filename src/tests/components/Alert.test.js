import React from 'react';
import { shallow } from 'enzyme';

import Alert from '../../app/components/Alert';

test('renders Alert component without crashing', () => {
  const props = { status: 'success', message: '' };

  const wrapper = shallow(<Alert {...props} />);
  expect(wrapper).toMatchSnapshot();
});
