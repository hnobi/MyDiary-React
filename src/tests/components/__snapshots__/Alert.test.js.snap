// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Alert component without crashing 1`] = `
<div>
  <Container
    position="top_center"
    store={
      Object {
        "data": Object {
          "classNames": undefined,
          "message": "",
          "status": "success",
          "timer": undefined,
        },
        "error": [Function],
        "info": [Function],
        "success": [Function],
        "unwatch": [Function],
        "warning": [Function],
        "watch": [Function],
      }
    }
  />
</div>
`;
